### PR TITLE
Master

### PR DESCRIPTION
**Pull request by msturrus**
Hi Matt,

Sorry this assignment is late but I commit from the github app itself
and it didn’t take e for some reason yesterday and I just noticed it
today.

My slide toggles do not work!  I looked everywhere for what I was doing
wrong and could not find it.

### CODE REVIEW

File: 08_Intro_jQuery/solution_student/select_fun/js/main.js [hunk 1]
```diff
@@ -30,11 +38,13 @@
 */
 
 // 6. Hide the item with the id #dropdownMenu
-
+$('#dropdownMenu').hide();
 
 // 7. When the element with the id #dropdownButton is clicked,
 	// a) slide toggle #dropdownMenu
-
+$('##dropdownButton').on('click', function() {
```
Comment (itsmattsoria) [inline 08_Intro_jQuery/solution_student/select_fun/js/main.js:45]: So close! You wrote it all right and had the correct logic, you just accidentally put two '#'s in your selector! That's why it didn't work: `$('##dropdownButton')`

File: 08_Intro_jQuery/solution_student/select_fun/js/main.js [hunk 2]
```diff
@@ -44,16 +54,22 @@
...
 // 9. Show the item with the id #answer1
-
+$('#answer1').show();
 
 // 10. When #question2 is clicked:
 	// a) Slide down #answer2
 	// b) Slide up #answer1
 	// c) Remove the active class from all list items
 	// d) Add the active class to #question2
+$('#question2').on('click', function() {
```
Comment (itsmattsoria) [inline 08_Intro_jQuery/solution_student/select_fun/js/main.js:67]: Great job on this one!

File: 08_Intro_jQuery/solution_student/select_fun/js/main.js [hunk 3]
```diff
@@ -7,21 +7,29 @@
...
+$('#box2').on('click', function() {
+	$('.box').removeClass('.crazy');
+	$('h1').addClass('crazy');
+	$('#box3').slideToggle();
+});
 
 // 4. When any .box is clicked, change the text in the header's h1 tag to read "jQuery Ninja"
-
-
+/**$('.box')on('click', function() {
```
Comment (itsmattsoria) [inline 08_Intro_jQuery/solution_student/select_fun/js/main.js:23]: You were so close on this one! You just forgot the period before the word 'on'! You wrote:

```
$('.box')on('click', function() {
```

but should have written:

```
$('.box').on('click', function() {
```

That's all!

**Comment by itsmattsoria** [comment]
Hey @msturrus, no problem! You did well, you just had some minor character errors, which I commented on in the code, overall good job though! The only thing I'm curious about is that I don't see the rest of the assignment, which was to do the select_fun_part2 in the assignment folder: https://github.com/GA-Chicago-FEWD6/Week_04_Intro_Programming/tree/master/Assignment
If you have a chance to go through that before tuesday I'd love you to submit another pull request. 
Hope you had a good weekend! :+1:

**Comment by msturrus** [comment]
I am very sorry Matt!  I must have missed the fun_part2 part of the
assignment.  I will have it done for Tuesday for sure.

**Comment by itsmattsoria** [comment]
@msturrus no problem!

**Pull request closed without merging**